src/Sulu/Bundle/AdminBundle/Resources/js/stores/snackbarStore/types.js

```diff
@@ -0,0 +1,6 @@
+// @flow
+export type Message = {
+    icon?: string,
+    text: string,
```

Comment:
think i would call this `message` as it is also called `message` on the `Snackbar` and in the `Overlay`

Reply:
find `message.message` strange.